packages/react-query/src/__tests__/suspense.test.tsx

```diff
@@ -876,9 +916,18 @@ describe("useQuery's in Suspense mode", () => {
     fireEvent.click(rendered.getByLabelText('fail'))
     // render error boundary fallback (error boundary)
     await waitFor(() => rendered.getByText('error boundary'))
+    if (consoleMock.mock.calls[0]) {
+      //test the error message that was passed to the console
+      expect(consoleMock.mock.calls[0][0]).toContain('Suspense Error Bingo')
+    }
```

Comment:
there is even a lint rule for that: [no-conditional-expect](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md)